modules/lib/default.nix

```diff
@@ -6,6 +6,7 @@ rec {
   assertions = import ./assertions.nix { inherit lib; };
 
   booleans = import ./booleans.nix { inherit lib; };
+  darwin = import ./darwin.nix { inherit lib; };
```

Comment:
Was wondering if this should be a top level uniqueness or if it makes sense to be an interval module with a `mkDarwin*` function instead

Reply:
I don't understand what you mean here.

Reply:
`lib.hm.interval` vs `lib.hm.darwin`